backend/api/EventHandlers/MissionEventHandler.cs

```diff
@@ -196,7 +196,6 @@ private async void OnSendRobotToDockTriggered(object? sender, RobotEmergencyEven
             catch (Exception ex)
             {
                 _logger.LogError(ex, "Failed to send robot {RobotId} to dock", robot.Id);
-                return;
```

Comment:
This could be nice to keep in case someone adds more code below later, but it's not too important

Reply:
Can reintroduce this one if that is preferred. Don't have a strong opinion on this specific one